source/agora/crypto/Key.d

```diff
@@ -43,10 +43,6 @@ unittest
     // Another keypair won't verify it
     KeyPair other = KeyPair.random();
     assert(!other.address.verify(sign, "Hello World".representation));
-
-    // Signature can't be changed
-    sign[].ptr[0] = sign[].ptr[0] ? 0 : 1;
-    assert(!kp.address.verify(sign, "Hello World".representation));
```

Comment:
??

Reply:
Reverted but changed the signature by signing something else as it didn't compile as it was.